server/handlers/authorization.go

```diff
@@ -79,7 +79,7 @@ func Authorize(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 		// load the authorize page
-		t, err := template.ParseFiles("assets-v1/templates/authorize.html")
+		t, err := template.ParseFiles("assets-v1/templates/src/pages/oauth_portal/authorize.html")
```

Comment:
Actually path to the folder oauth_portal itself can be extracted, so there would be no need to copy-paste it every time.